### PR TITLE
fix(HACBS-1575): change errors to warnings

### PR DESCRIPTION
**Pull request by scoheb**
The script can be more permissible when detecting SPI phases. We have converted Errors to be warnings

### CODE REVIEW

File: setup-quay-push-secret.sh [hunk 1]
```diff
@@ -77,8 +75,7 @@ echo "Waiting for Injected phase"
 kubectl wait --for=jsonpath='{.status.phase}'=Injected Spiaccesstokenbinding/${binding_name} -n $target_namespace --timeout=60s
 
 if [ $? -ne 0 ] ; then
-  echo "Error: could verify if secret was injected after 60s. Verify your input."
-  exit 1
+  echo "warning: could verify if secret was injected after 60s. Verify your input."
```
Comment (davidmogar) [inline setup-quay-push-secret.sh:78]: Is it `could` or `couldn't`?

File: setup-quay-push-secret.sh [hunk 2]
```diff
@@ -57,13 +57,11 @@ spec:
...
 EOF
 echo "Binding created"
-sleep 3
 echo "Waiting for AwaitingTokenData phase"
 kubectl wait --for=jsonpath='{.status.phase}'=AwaitingTokenData Spiaccesstokenbinding/${binding_name} -n $target_namespace --timeout=60s
 
 if [ $? -ne 0 ] ; then
-  echo "Error: could get token data after 60s. Verify your input."
-  exit 1
+  echo "warning: could get token data after 60s. Verify your input."
```
Comment (davidmogar) [inline setup-quay-push-secret.sh:64]: Is it `could` or `couldn't`?

**Review by davidmogar — COMMENTED** [review]
Please, add a description to the PR and to the commit.

File: setup-quay-push-secret.sh (hunk 1 above)
Comment (scoheb) [reply setup-quay-push-secret.sh]: fixed

**Review by happybhati — APPROVED** [review]

**Pull request merged**